src/configuration/logger.py

```diff
@@ -0,0 +1,85 @@
+import os
+import logging
+import sys
+sys.path.append("../utils")
+from singleton import Singleton
+
+
+class LoggerAdmin(object):
+
+    __metaclass__ = Singleton
+
+    def __init__(self,log_file_path, log_file_name="logger.log"):
+        """
+        Initialize attibutes
+        log_file_path -- the path where .log file is stored,
+        log_file_name -- the name of .log file by default logger        
+        """
+        self.log_file_path = log_file_path
+        file_handler = self.get_file_handler(log_file_name)
+        self.logger = logging.getLogger()
+        self.logger.setLevel(logging.DEBUG)
+        formatter = logging.Formatter("%(asctime)s %(name)s - %(levelname)s - %(message)s")
+        file_handler.setFormatter(formatter)
+        self.logger.addHandler(file_handler)
+
+    def get_file_handler(self, log_file_name):
+        """
+        The method will convert the log_file_name in a absolute path
+        log file name -- name of log file
+        """
+        self.log_file_abs_path = os.path.abspath(self.log_file_path + log_file_name)
+        filehandler = logging.FileHandler(self.log_file_abs_path, 'a')
```

Comment:
Review the variable names to be consistent you have filehandler and file_handler